src/main/webapp/components/CustomMenu.jsx

```diff
@@ -0,0 +1,70 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { _ } from '@splunk/ui-utils/i18n';
+import { getUnifiedConfigs } from '../util/util';
+
+class CustomMenu extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            loading: true,
+        };
+        this.shouldRender = true;
+    }
+
+    componentDidMount() {
+        const globalConfig = getUnifiedConfigs();
+        this.setState({ loading: true });
+        this.loadCustomControl().then((Control) => {
+            const customControl = new Control(globalConfig, this.el, this.setValue);
```

Comment:
Can we change name of **CustomControl** to **CustomMenu** everywhere?

Reply:
Done